native/acter/src/api/attachments.rs

```diff
@@ -361,6 +362,26 @@ impl AttachmentsManager {
         })
     }
 
+    pub fn location_attachment_draft(
+        &self,
+        body: String,
+        geo_uri: String,
+    ) -> Result<AttachmentDraft> {
+        if !self.is_joined() {
+            bail!("Can only attachment in joined rooms");
+        }
+        let mut builder = self.inner.draft_builder();
```

Comment:
I think we would also want to add line for room member post permission check here too?

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1196/commits/dff6d53fbca074513d5a396cb3eb4bc1eb40f256